.github/workflows/scalability.yml

```diff
@@ -0,0 +1,24 @@
+name: Scalability Workflow
+# This workflow is triggered on pushes to the repository.
+on:
+  push:
+    branches: 
+      - scalability-automation
```

Comment:
Any thoughts on when we should trigger this? Is there a manual trigger option? Or maybe trigger it when we push a beta tag...

Reply:
I was thinking to just trigger immediately when the beta tag gets created. There is an option to make it manual with github actions with minimal effort.

https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/